src/sentry/notifications/utils/__init__.py

```diff
@@ -623,6 +630,46 @@ def parallelizable_spans(self) -> List[str]:
     def _find_span_desc_by_id(self, id: str) -> str:
         return get_span_evidence_value(self._find_span_by_id(id))
 
+    @property
+    def duration_impact(self) -> str:
+        transaction_duration = self.transaction_duration
+        time_saved = self._calculate_time_saved()
+        time_saved_percentage = time_saved / transaction_duration * 100
+        return (
+            f"{round(time_saved_percentage,3)}% ({int(time_saved)}ms/{int(transaction_duration)}ms)"
+        )
+
+    def _sum_span_duration(self, spans: list) -> int:
+        "Given a non-overlapping spans, find the sum of the span durations in milliseconds"
+        sum = 0
+        for span in spans:
+            sum += self.get_span_duration(span).total_seconds() * 1000
+        return sum
```

Comment:
Nice one! Probably good to put into `performance_issues/base.py` this is a nice helper